### ISSUE TITLE
Collection support

### ISSUE THREAD

**Issue by tagirb** [doy/rbw#128]
Thanks for rbw! Seems to be much faster than the official Bitwarden Client. However, does it support collections? Can't seem to find it..

**Comment by doy** [comment]
no, i don't really use organizations much, so a lot of related features aren't yet implemented. if there are specific things you need, i'd be happy to look over pull requests though.

**Comment by tagirb** [comment]
No problem, thanks! I just discovered that Bitwarden now offers [Bitwarden Secrets Manager](https://bitwarden.com/help/secrets-manager-quick-start/), which also has Rust-based client and targeted exactly at our use case.
